go.mod

```diff
@@ -5,48 +5,45 @@ go 1.14
 replace (
 	github.com/hashicorp/consul v0.0.0-20171026175957-610f3c86a089 => github.com/hashicorp/consul/sdk v0.7.0
 	github.com/hashicorp/consul/api v1.3.0 => github.com/hashicorp/consul/api v0.0.0-20191112221531-8742361660b6
+	github.com/kyma-incubator/hydroform/function => github.com/pPrecel/hydroform/function v0.0.0-20210206085647-39a02f919fe9
```

Comment:
Please remove this replace directive.

Reply:
I can't because my second PR is still in review. I will change it later (before merge)

Reply:
I thought so yes, just a reminder so that we do not forget.